### PR TITLE
feat(ci): enable garnix on-push evaluation

### PR DESCRIPTION
**Pull request by PoisonPhang**

### CODE REVIEW

**Review by benluelo — DISMISSED** [review]

File: flake.nix [hunk 1]
```diff
@@ -475,7 +475,11 @@
               checkPhase = ''
                 cd $src/.
                 for i in `find . -name "*.nix" -type f`; do
-                    nil diagnostics "$i"
+                    if [[ $i =~ 'vendor' ]]; then
+                      echo "skipping $i"
+                    else
+                      nil diagnostics "$i"
+                    fi
```
Comment (KaiserKarel) [inline flake.nix:482]: why not do this in the src filter?

Comment (PoisonPhang) [reply flake.nix:482]: Good point

**Review by benluelo — APPROVED** [review]

File: flake.nix (hunk 1 above)
Comment (PoisonPhang) [reply flake.nix]: 8d4fbb2dd26f3b8ffd4bf6ea40973e794f6a0446

**Review by KaiserKarel — APPROVED** [review]
lgtm

**Pull request merged**